api/src/main/java/org/eclipse/microprofile/lra/annotation/Status.java

```diff
@@ -35,40 +36,86 @@
  * and value is of type {@link java.net.URI}.
+ * </p>
  *
+ * <p>
  * If the annotated method is not a JAX-RS resource method the id of the currently
  * running LRA can be obtained by adhering to a predefined method signature as
  * defined in the LRA specification document. Similarly the method may determine
  * whether or not it runs with a nested LRA by providing a parameter to hold the parent id.
  * For example,
+ * </p>
+ *
  * <pre>
  *     <code>
  *          &#64;Status
  *          public void status(URI lraId, URI parentId) { ...}
  *     </code>
  * </pre>
+ *
+ * <p>
  * would be a valid status method declaration. If an invalid signature is detected 
  * the implementation of this specification MUST prohibit successful startup of the application
  * (e.g. with a runtime exception).
+ * </p>
  *
+ * <p>
  * If the participant has already responded successfully to an invocation
- * of the <em>@Compensate</em> or <em>@Complete</em> method then it may
- * report <em>404 Not Found</em> HTTP status code or in case of 
- * non-JAX-RS method returning {@link ParticipantStatus} to return <em>null</em>. 
+ * of the <code>&#64;Compensate</code> or <code>&#64;Complete</code> method then it may
+ * report <code>404 Not Found</code> HTTP status code or in case of
+ * non-JAX-RS method returning {@link ParticipantStatus} to return <code>null</code>.
  * This enables the participant to free up resources.
+ * </p>
  *
+ * <p>
  * Since the participant generally needs to know the id of the LRA in order
  * to report its status there is generally no benefit to combining this
- * annotation with the `@LRA` annotation (though it is not prohibited).
+ * annotation with the <code>&#64;LRA</code> annotation (though it is not prohibited).
+ * </p>
+ *
+ * <p>
+ * If the method is a JAX-RS resource method (or is a non JAX-RS method
+ * annotated with <code>&#64;Status</code> with return type
+ * <code>javax.ws.rs.core.Response</code> then the following are the only
+ * valid response codes:
+ * </p>
+ *
+ * <table border="0" cellpadding="3" cellspacing="0"
+ *   summary="Valid JAX-RS response codes for Status methods">
+ * <caption><span>JAX-RS Response Codes For Status Methods</span><span>&nbsp;</span></caption>
+ * <tr>
+ *   <th scope="col">Code</th>
+ *   <th scope="col">Response Body</th>
+ *   <th scope="col">Meaning</th>
+ * </tr>
+ * <tr>
+ *   <td scope="row">200</td>
+ *   <td scope="row">{@link ParticipantStatus} enum value</td>
+ *   <td scope="row">The current status of the participant</td>
+ * </tr>
+ * <tr>
+ *   <td scope="row">202</td>
+ *   <td scope="row">Empty</td>
+ *   <td scope="row">The resource is attempting to determine the status and
+ *   the caller should retry later</td>
+ * </tr>
+ * <tr>
+ *   <td scope="row">404</td>
+ *   <td scope="row">Empty</td>
+ *   <td scope="row">The method does not know about the LRA</td>
+ * </tr>
+ * </table>
```

Comment:
we can also add something like 

```
<p>
 * If any other code is returned then the implementation SHOULD keep retrying.
 * If it stops retrying a warning message SHOULD be logged.
 * </p>
```

Reply:
+1

Reply:
Actually I don't think including this text has any meaning since it is covered implicitly in the javadoc for Compensate/Complete. If you don't mind I will back out this paragraph.

Reply:
There are no restrictions on how often the Status method can be called so adding the text does not add anything to the spec.

Reply:
Note that if the status cannot be obtained and the implementation stops retrying then the javadoc for Compensate/Complete already indicates that a warning will be logged (ie if we added this text two warnings would be logged).

Reply:
I don't mind not mentioning it here, it is kind of implicit. Can be resolved.